src/main/java/net/unicon/lti/controller/lti/RegistrationController.java

```diff
@@ -193,40 +209,38 @@ private ToolRegistrationDTO generateToolConfiguration() {
         //OPTIONAL -->setTos_uri
         //OPTIONAL -->setPolicy_uri
         ToolConfigurationDTO toolConfigurationDTO = new ToolConfigurationDTO();
-        toolConfigurationDTO.setDomain(localUrl.substring(localUrl.indexOf("//") + 2));
+        toolConfigurationDTO.setDomain(domainUrl);
```

Comment:
Are we limited to a single domain url, then?  ie, can we only do dynamic registration through Goldi (and if we had a second service we'd like to do dynamic registration through, we'd need to set up a second instance of the middleware)?

Reply:
The Dynamic Registration specification mentions a field called secondary_domains in which additional domains may be included. Since this specification is so new though, I'm not sure if D2L and Moodle support that field or not. From what I've seen, the standard registration process doesn't appear to include the ability to have multiple domains (which makes me think that the LMSs also wouldn't support it for Dynamic Registration), but I could do more research to confirm that if you'd like.

Reply:
I don't think further research on that is necessary right now. Until the spec is officially and more widely supported, I believe we're good for the time being - thank you!